ksqldb-functional-tests/src/test/resources/query-validation-tests/greatest.json

```diff
@@ -0,0 +1,178 @@
+{
+  "comments": [
+    "Tests covering the greatest function."
+  ],
+  "tests": [
+    {
+      "name": "test greatest with numeric types",
+      "statements": [
+        "CREATE STREAM INPUT1 (ID BIGINT KEY, I1 INT, I2 INT, I3 INT) WITH (kafka_topic='input_topic_1', value_format='json');",
+        "CREATE STREAM INPUT2 (ID BIGINT KEY, B1 BIGINT, B2 BIGINT, B3 BIGINT) WITH (kafka_topic='input_topic_2', value_format='json');",
+        "CREATE STREAM INPUT3 (ID BIGINT KEY, DE1 DECIMAL(9,3), DE2 DECIMAL(9,3), DE3 DECIMAL(9,3)) WITH (kafka_topic='input_topic_3', value_format='json');",
+        "CREATE STREAM INPUT4 (ID BIGINT KEY, DO1 DOUBLE, DO2 DOUBLE, DO3 DOUBLE) WITH (kafka_topic='input_topic_4', value_format='json');",
+
+        "CREATE STREAM OUTPUT1 AS SELECT ID, GREATEST(I1, I2, I3, null, 5) AS HIGHEST FROM INPUT1;",
```

Comment:
The QTT framework does not support multiple persistent queries like this. (For example, you can see that the spec.json file generated for this test only recorded schemas for the last one. The topology file only captures the last one as well.) This test needs to be split into four separate QTT tests.

(Same for the analogous test for LEAST below.)